src/test/java/com/redislabs/redisai/ChunkTest.java

```diff
@@ -0,0 +1,55 @@
+package com.redislabs.redisai;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Ignore;
+import org.junit.Test;
+
+@Ignore
+public class ChunkTest {
+
+  private static final int SMALL_CHUNK_SIZE = 8 * 1024; // 8KB
+
+  @BeforeClass
+  public static void prepare() {
+    System.setProperty(Model.BLOB_CHUNK_SIZE_PROPERTY, Integer.toString(SMALL_CHUNK_SIZE));
+  }
+
+  @AfterClass
+  public static void cleanUp() {
+    System.clearProperty(Model.BLOB_CHUNK_SIZE_PROPERTY);
+  }
+
+  /**
+   * @throws java.net.URISyntaxException
+   * @throws java.io.IOException
+   * @see ModelTest#argumentsWithoutChunking()
+   */
+  @Test
+  public void argumentsWithChunking() throws URISyntaxException, IOException {
+    Model model =
+        new Model(
+            Backend.ONNX,
+            Device.GPU,
+            getClass().getClassLoader().getResource("test_data/mnist.onnx").toURI());
+
+    Assert.assertEquals(8, model.getModelStoreCommandArgs("key").size());
+  }
+
+  @Test
+  public void commandWithChunking() throws IOException, URISyntaxException {
+    Model model =
+        new Model(
+            Backend.ONNX,
+            Device.CPU,
+            getClass().getClassLoader().getResource("test_data/mnist.onnx").toURI());
+
+    try (RedisAI ai = new RedisAI()) {
```

Comment:
@sazzad16 can you please verify that the loaded model is still functional after it was loaded?